store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedGraphStorageTraitsTest.java

```diff
@@ -174,59 +179,79 @@ public void shouldVerifyAssumptionsNoTraitsFound() throws Exception {
-        Set<StoreTrait> intersectionTraitsIsCurrent = accTraitsIsCurrent.stream().filter(mapTraitsIsCurrent::contains).collect(Collectors.toSet());
-        Set<StoreTrait> mapTraitsIsCurrentIsSubSetOfStoreTraits = mapTraitsIsCurrent.stream().filter(t -> !MAP_TRAITS.contains(t)).collect(Collectors.toSet());
-        Set<StoreTrait> accTraitsIsCurrentIsSubSetOfStoreTraits = accTraitsIsCurrent.stream().filter(t -> !ACCUMULO_TRAITS.contains(t)).collect(Collectors.toSet());
-
-        //then
+        final Set<StoreTrait> mapTraitsIsCurrent = map.getGraph().execute(getTraits, testUser);
+        final Set<StoreTrait> accTraitsIsCurrent = acc.getGraph().execute(getTraits, testUser);
+
+        // when
+        final Set<StoreTrait> mapTraitsIsCurrentExclusive = mapTraitsIsCurrent.stream()
+                .filter(t -> !accTraitsIsCurrent.contains(t)).collect(Collectors.toSet());
+        final Set<StoreTrait> accTraitsIsCurrentExclusive = accTraitsIsCurrent.stream()
+                .filter(t -> !mapTraitsIsCurrent.contains(t)).collect(Collectors.toSet());
+        final Set<StoreTrait> intersectionTraitsIsCurrent = accTraitsIsCurrent.stream().filter(mapTraitsIsCurrent::contains)
+                .collect(Collectors.toSet());
+        final Set<StoreTrait> mapTraitsIsCurrentIsSubSetOfStoreTraits = mapTraitsIsCurrent.stream()
+                .filter(t -> !MAP_TRAITS.contains(t)).collect(Collectors.toSet());
+        final Set<StoreTrait> accTraitsIsCurrentIsSubSetOfStoreTraits = accTraitsIsCurrent.stream()
+                .filter(t -> !ACCUMULO_TRAITS.contains(t)).collect(Collectors.toSet());
+
+        // then
         assertNotEquals(ACCUMULO_TRAITS, accTraitsIsCurrent);
         assertNotEquals(MAP_TRAITS, mapTraitsIsCurrent);
-        assertEquals(ACC_CURRENT_TRAITS, accTraitsIsCurrent, "Expected traits for the AccumuloStore 'Current schema' is different");
-        assertEquals(MAP_CURRENT_TRAITS, mapTraitsIsCurrent, "Expected traits for the MapStore 'Current schema' is different");
-        assertEquals(Collections.emptySet(), mapTraitsIsCurrentExclusive, "Expected traits exclusive to MapStore is different");
-        assertEquals(Sets.newHashSet(StoreTrait.ORDERED), accTraitsIsCurrentExclusive, "Expected traits exclusive to AccumuloStore is different");
-        assertEquals(Sets.newHashSet(StoreTrait.INGEST_AGGREGATION, StoreTrait.MATCHED_VERTEX, StoreTrait.PRE_AGGREGATION_FILTERING, StoreTrait.TRANSFORMATION, StoreTrait.POST_AGGREGATION_FILTERING, StoreTrait.POST_TRANSFORMATION_FILTERING), intersectionTraitsIsCurrent, "Expected  intersection traits is different");
-        assertEquals(Collections.emptySet(), mapTraitsIsCurrentIsSubSetOfStoreTraits, "The IsCurrent traits is not a subset of MapStore traits");
-        assertEquals(Collections.emptySet(), accTraitsIsCurrentIsSubSetOfStoreTraits, "The IsCurrent traits is not a subset of AccumuloStore traits");
+        assertEquals(ACC_CURRENT_TRAITS, accTraitsIsCurrent,
```

Comment:
👎  You've inserted a lot of line wrap which makes spot the difference harder to see for these tests in future.
```
xxxxx xxxx xx MapTraits xxxx xxx x xx
xxxxx xxxx xx AccTraits xxxx xxx x xx
```